app/models/moirai/key_finder.rb

```diff
@@ -12,16 +12,22 @@ def initialize
     end
 
     # TODO: remove locale default
-    def file_path_for(key, locale: I18n.locale)
+    # Returns all the file_paths where the key is found, including gems.
+    def file_paths_for(key, locale: I18n.locale)
+      return [] if key.blank?
+
       locale ||= I18n.locale
-      moirai_translations[locale.to_sym][key]
+      moirai_translations[locale.to_sym].select do |_filename, data|
```

Comment:
Why not `filter_map`?
```suggestion
      moirai_translations[locale.to_sym].filter_map { |filename, data| filename if data.dig(*key.split(".")).present? }
  .sort_by { |file_path| file_path.start_with?(Rails.root.to_s) ? 0 : 1 }
```